core/src/main/java/bio/terra/pearl/core/model/study/Study.java

```diff
@@ -16,6 +16,7 @@
 public class Study extends BaseEntity {
     private String name;
     private String shortcode;
+    private String pepperStudyName;
```

Comment:
oh, this is an actual field -- did you mean to have the dao method search on this instead?